pkg/k8s/util.go

```diff
@@ -184,7 +187,25 @@ func DCRackLabelsAndNodeSelectorForStatefulSet(cc *api.CassandraCluster, dc int,
 	return labels, nodeSelector
 }
 
+// LookupCassandra Cluster returns the running cluster instance based on its name and namespace
+func LookupCassandraCluster(client runtimeClient.Client, clusterName, clusterNamespace string) (cluster *api.CassandraCluster, err error) {
+	cluster = &api.CassandraCluster{}
+	err = client.Get(context.TODO(), types.NamespacedName{Name: clusterName, Namespace: clusterNamespace}, cluster)
```

Comment:
just return here

Reply:
I can't because I have to setup err value before, I just don't return err, but also cluster var.